package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ionic-native-http-connection-backend",
-  "version": "0.0.0-development",
-  "description": "A solution to CORS problem with Ionic 3 and WKWebView",
+  "version": "4.0.6",
```

Comment:
The new version of the plugin is being created and published by CI with the help of https://github.com/semantic-release/semantic-release

The latest versing could be bumped into package.json by CI as well. With the help of this plugin https://github.com/semantic-release/git

What is the reason behind putting version into package.json ?

Reply:
It is not required because of the build process but it would be "nice to have".